### PR TITLE
Add ability to reply to a message

### PR DESCRIPTION
**Pull request by Johnnycus**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3433,63 +3433,177 @@ <h1 id="title" class="text-xl font-bold">Omnispace</h1>
...
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      d="M9 15L3 9m0 0l6-6M3 9h12a6 6 0 010 12h-3"
+                    ></path>
+                  </svg>
+
+                  <div class="flex flex-col pl-2 w-[calc(100%-30px)]">
+                    <p x-text="senderName" class="font-bold"></p>
+                    <p x-text="messageContent" class="truncate"></p>
+                  </div>
```
Comment (billythedummy) [inline index.html:3555]: this part doesnt seem to show up when i try to reply to a reply

Comment (Johnnycus) [reply index.html:3555]: any errors in the console?

Comment (billythedummy) [reply index.html:3555]: did removing the `<pre>` have something to do with it?

**Review by billythedummy — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (Johnnycus) [reply index.html:3557]: <pre> was there just for seeing the data of `replyingToMessage` object, shouldn't have affect anything

Comment (Johnnycus) [reply index.html:3557]: lack of hot module replacement and these cache issues is super annoying

**Pull request merged**